tf2_geometry_msgs/src/tf2_geometry_msgs/tf2_geometry_msgs.py

```diff
@@ -62,6 +63,95 @@ def transform_to_kdl(t):
                                     t.transform.translation.y,
                                     t.transform.translation.z))
 
+def transformCovariance(cov_in, transform):
```

Comment:
Nit: the style in this file is snake_case, so I'd prefer this to be `transform_covariance`.

Reply:
Right, done!